### ISSUE TITLE
Failed to verify username/invalid session

### ISSUE THREAD

**Issue by NebulaBC** [haveachin/infrared#140]
**Describe the bug**
Recently all of my servers have started sending an invalid session error. I have tried numerous things including running a similar proxy (Ultraviolet) and the error persists. I can run a proxy from a separate machine and it will work perfectly. I'm curious if Mojang has implemented something to do with connecting from the same machine? (since infrared is on the same machine)
I have reset my configs, gone back to older versions. I have tried Paper servers that are not running my custom fork, Spigot, and Fabric servers and all of them are giving this error. There is no error on the proxy side but the server just gives the error and kicks the player. This issue is not just happening for me either, I have had the issue happen with nearly 20 people (everyone who has tried to join) from all over the world.

**To Reproduce**
What OS are you using?
Ubuntu 20.04 ARM64
What version of Infrared are you using?
I have tried the latest version and older versions ranging all the way back to 1.2 I have also tried the fork by lhridder and other various proxies.

**What do your Infrared proxy config(s) look like?**
I have changed my configs to experiment, and have had the issue persist no-matter what config I use including the provided minimal config.

**How do you trigger the bug/issue?**
Just use the proxy with a recent version of any server software. I am just baffled at how this could happen because it did not start when I updated any servers or anything, just out of the blue after not interacting with the VPS at all.

**Expected behavior**
Standard operation of the proxy

**Additional context** (Optional)
I assume this is either something Mojang changed with their auth servers, or I am just having an issue with my machine. I thought I would at least give it a go reporting this though in-case anyone had any ideas that I did not.

**Comment by lhridder** [comment]
Hi, are you running with prevent-proxy-connections enabled in server.properties on the backend by any chance?

**Comment by NebulaBC** [comment]
> Hi, are you running with prevent-proxy-connections enabled in server.properties on the backend by any chance?

Oh my god. I disabled prevent-proxy-connections and it instantly worked. I am at a loss, because this setting has been enabled since I first started using Infrared. I am not sure what happened, maybe I changed some internal network stuff without thinking? Thank you for resolving this. I feel like an idiot for going through all that troubleshooting without trying to disable that. 😂
